apps/meteor/server/models/raw/Roles.ts

```diff
@@ -179,6 +179,15 @@ export class RolesRaw extends BaseRaw<IRole> implements IRolesModel {
 		return this.find(query, options || {});
 	}
 
+	findCustomUserRoles(options?: FindOptions<IRole>): FindCursor<IRole> {
+		const query: Filter<IRole> = {
+			scope: 'Users',
```

Comment:
I'm wondering if we should use only "Users" scoped roles for the stats. Maybe we'd like to count both ("Users" and "Rooms") scopes or split them in two different stats. Let's check that.